.pipelines/custom-templates/post-publish-template.yml

```diff
@@ -0,0 +1,53 @@
+parameters:
+  - name: name
+    type: string
+  - name: libName
+    type: string
+  - name: path
+    type: string
+    default: 'lib'
+  - name: cdn
+    type: boolean
+    default: false
+
+steps:
+  - task: DownloadPipelineArtifact@2
+    condition: always()
+    inputs:
+      buildType: 'specific'
+      project: 'a7934fdd-dcde-4492-a406-7fad6ac00e17'
```

Comment:
Is there a variable we can use here instead?

Reply:
This was auto-populated by the ADO UI when selecting the specific pipeline from a drop-down. I haven't been able to find the place where the project and build reference actually live so I don't think we should mess with the value.